docs/en/operations/settings/settings.md

```diff
@@ -365,7 +365,20 @@ throws an exception.
 
 ## input_format_null_as_default {#settings-input-format-null-as-default}
 
-Enables or disables using default values if input data contain `NULL`, but the data type of the corresponding column in not `Nullable(T)` (for text input formats).
+Enables or disables insertion of default values if input data contain `NULL`, but the data type of the corresponding column in not `Nullable(T)` (for text input formats). This setting is applied to `INSERT ... VALUES` queries.
```

Comment:
```suggestion
For text input formats initialize `NULL` fields with default values if data type of this field is not nullable.
```

Reply:
I mentioned `INSERT ... VALUES` query to clear the difference between **input_format_null_as_default** and **insert_null_as_default** settings.
Can we leave your variant and append _This setting is applied (or applicable) to INSERT ... VALUES queries_ to it?

Reply:
What does _For text input formats_ mean?
It seems that this setting is also used for numeric data types. I have checked Int32:
```
CREATE TABLE test_null_as_default (a String DEFAULT '_', b Int32 DEFAULT 99) ENGINE = Memory;

SET input_format_null_as_default=1;
INSERT INTO test_null_as_default (a, b) VALUES ('A0', 10), (NULL, NULL);
SELECT * FROM test_null_as_default;
```
```
A0      10
_       99
```

Reply:
> Can we leave your variant and append

yes, ok.

> What does For text input formats mean?

It means CSV, TSV, etc.